### PR TITLE
Updated search order for ARM libraries.

### PR DESCRIPTION
**Pull request by mnlipp**
Fixes https://github.com/NeuronRobotics/nrjavaserial/issues/47.

### CODE REVIEW

File: src/main/java/gnu/io/NativeResource.java [hunk 1]
```diff
@@ -139,7 +139,21 @@ private void loadResource(File resource) {
 		if(!resource.canRead())
 			throw new RuntimeException("Cant open JNI file: "+resource.getAbsolutePath());
 		//System.out.println("Loading: "+resource.getAbsolutePath());
+		try {
+			System.out.println("Going to load " + resource + ". Hit return...");
```
Comment (kaikreuzer) [inline src/main/java/gnu/io/NativeResource.java]: I don't think anybody wants `System.out` being used in the productive code?

File: src/main/java/gnu/io/NativeResource.java [hunk 2]
```diff
@@ -139,7 +139,21 @@ private void loadResource(File resource) {
...
+			System.in.read();
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```
Comment (kaikreuzer) [inline src/main/java/gnu/io/NativeResource.java]: Similar comment as above: e.printStackTrace(); should imho not be used.

File: src/main/java/gnu/io/NativeResource.java (hunk 1 above)
Comment (mnlipp) [inline src/main/java/gnu/io/NativeResource.java]: Sorry, of course I wanted to submit only the change of line 43. I had forgotten about the extra debug code still being there and it wasn't in my editor windows (those are always too small) so I overlooked it when committing.

File: src/main/java/gnu/io/NativeResource.java (hunk 2 above)
Comment (mnlipp) [inline src/main/java/gnu/io/NativeResource.java]: See above.

**Comment by MrDOS** [comment]
LGTM.

**Pull request merged**